examples/kelvin_helmholtz_instability.jl

```diff
@@ -35,17 +35,18 @@ B = BackgroundField(stratification, parameters=(Ri=0.1, h=1/4))
 
 using Plots, Oceananigans.Grids
 
-z = znodes(Cell, grid)
+zF = znodes(Face, grid)
+zC = znodes(Cell, grid)
```

Comment:
💯

Reply:
for plotting the use of Plots Recipes will do the job...